docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/azure.md

```diff
@@ -90,7 +90,7 @@ To create a security group that enables these artifacts, navigate to **Network S
 * In the Name field, enter yugaware-sg or any other name.
 * Edit inbound security rules, as follows:
   * Add the appropriate IP addresses to the **Source IP ranges** field. To allow access from any machine, enter 0.0.0.0/0, but note that this is not very secure.
-  * Add the ports 22, 8800, and 80 to the **Port Range** field. The protocol selected must be TCP. For a self-managed configuration, also add 7000, 7100, 9000, 9100, 11000, 12000, 9300, 9042, 5433, and 6379 to the **Port Range** field.
+  * Add the ports 22, 8800, 80, and 18018 to the **Port Range** field. The protocol selected must be TCP. For a self-managed configuration, also add 7000, 7100, 9000, 9100, 11000, 12000, 9300, 9042, 5433, and 6379 to the **Port Range** field.
```

Comment:
Rewrite it as:

Add the ports 22, 8800, and 80 to the **Port Range** field. The protocol selected must be TCP. For a self-managed configuration, also add 7000, 7100, 9000, 9100, 18018, 11000, 12000, 9300, 9042, 5433, and 6379 to the **Port Range** field.